web/src/main/kotlin/no/nav/su/se/bakover/web/routes/behandling/BehandlingsinformasjonJson.kt

```diff
@@ -183,11 +181,11 @@ internal fun Behandlingsinformasjon.OppholdIUtlandet.toJson() =
         begrunnelse = begrunnelse
     )
 
-internal fun Behandlingsinformasjon.Formue.toJson() =
+internal fun Behandlingsinformasjon.Formue.toJson(ektefellePartnerSamboer: Behandlingsinformasjon.EktefellePartnerSamboer?) =
     FormueJson(
         status = status.name,
         verdier = this.verdier?.toJson(),
-        borSøkerMedEPS = this.borSøkerMedEPS,
+        borSøkerMedEPS = (ektefellePartnerSamboer as? Behandlingsinformasjon.EktefellePartnerSamboer.Ektefelle) != null,
```

Comment:
Skal ikke trenge 'Safe cast operator' (`as?`), hvis du er garantert att få in den riktiga typen. Ser ju ut som att man borde få det i parametern.

Reply:
Behandlingsinformasjon.EktefellePartnerSamboer? kan ha 3 tilstander (null | Ektefelle | IngenEktefelle). Vi har ikke noe grunnlag for å vite hvilken av de 3 tilstandene den er i.